.github/workflows/main.yml

```diff
@@ -93,24 +129,29 @@ jobs:
           - dbt-core:
               version: head
```

Comment:
Just curious what's the difference between HEAD and main for dbt core, will core point HEAD other than main?

Reply:
No difference, `head` will just point to main in core. The `version` noted here is used to build the image name, and we want them named with `head`. The `main` branch referenced in the matrix is for dbt-server rather than core